web/js/data/results.js

```diff
@@ -1,45 +1,59 @@
-/**
- * @module wv.data
- */
-var wv = wv || {};
-wv.data = wv.data || {};
-
-wv.data.results = wv.data.results || {};
-
-wv.data.results.antiMeridianMulti = function (maxDistance) {
+import $ from 'jquery';
+
+import OlGeomMultiPolygon from 'ol/geom/multipolygon';
+import OlGeomMultiPoint from 'ol/geom/multipoint';
+import OlFormatGeoJSON from 'ol/format/geojson';
+import OlGeomPolygon from 'ol/geom/polygon';
+import OlGeomPoint from 'ol/geom/point';
+import olExtent from 'ol/extent';
+import loClone from 'lodash/clone';
+import loEach from 'lodash/each';
+import {dataCmrRoundTime, dataCmrGeometry} from './cmr';
+import util from '../util/util';
+import {dataHelper} from '../edsc';
```

Comment:
Why not rename the whole `edsc` directory to `web/data/helpers` or something and eliminate the edsc name completely?